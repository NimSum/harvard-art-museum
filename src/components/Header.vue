<template>
  <header>
    <h1><span>Harvard Art</span>Musem Explorer</h1>
    <ul>
      <li>Home</li>
      <li>Floor<span>1</span></li>
      <li>Floor<span>2</span></li>
      <li>Floor<span>3</span></li>
      <li>Floor<span>4</span></li>
      <li>Floor<span>5</span></li>
    </ul>
  </header>
</template>

<script>
export default {
  name: 'Header'
}
</script>

<style lang="scss" scoped>
  header {
    display: flex;
    justify-content: space-between;
    background: #252A34;
    color: #E3FDFD;
  }

  h1 {
    display: flex;
    flex-direction: column;
    font-family: 'Rufina', serif;
    margin: 0 10px;
  }

  ul {
    display: flex;
    margin: 0;
    li {
      list-style-type: none;
      font-size: 1.2rem;
      padding: 0 20px;
      display: flex;
      justify-content: center;
      flex-direction: column;
      font-family: 'Scada', sans-serif;
      
      span {
        display: block;
      }
    }
  }
</style>
