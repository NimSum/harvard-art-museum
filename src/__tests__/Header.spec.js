import { mount, RouterLinkStub } from '@vue/test-utils';
import Header from '../components/Header.vue';

describe('Header', () => {
  test('renders correctly', () => {
    const wrapper = mount(Header, {
      stubs: {
        RouterLink: RouterLinkStub
      }
    });
    expect(wrapper.element).toMatchSnapshot();
  })
});