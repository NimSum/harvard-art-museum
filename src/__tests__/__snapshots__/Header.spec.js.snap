// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <a>
    <h1>
      <span>
        Harvard Art
      </span>
      Musem Explorer
    </h1>
  </a>
   
  <ul>
    <li>
      <a>
        
          Home
      
      </a>
    </li>
     
    <li>
      <a>
        
         Floor
        <span>
          1
        </span>
      </a>
    </li>
     
    <li>
      <a>
        
        Floor
        <span>
          2
        </span>
      </a>
    </li>
     
    <li>
      <a>
        
        Floor
        <span>
          3
        </span>
      </a>
    </li>
     
    <li>
      <a>
        
        Floor
        <span>
          4
        </span>
      </a>
    </li>
  </ul>
</header>
`;
